scripts/switchBranches.fish

```diff
@@ -5,14 +5,14 @@ function checkoutRepo
       echo "Checkout needs two parameters branch force"
       return 1
   end
-  set -l branch $argv[1]
+  set -l branch (string trim $argv[1])
```

Comment:
:+1: this change will reduce the number of required builds by approx. 10-20%!!! nice